### ISSUE TITLE
Générer slides intro ?

### ISSUE THREAD

**Issue by loicknuchel** [HumanTalks/humantalks-admin#47]

**Comment by sbouaked** [comment]
Ca serai top ! Faut qu'on sorte de slides.com pour les faire directement en HTML.

**Comment by loicknuchel** [comment]
Yep... après à voir si ça ne limite pas trop et comment les modifier et tout...
Eventuellement trouver un outil où on peut injecter des données ou créer une iframe pour la liste des talks et le lieu...
Enfin, à voir mais mieux vaut pas trop se presser là dessus, c'est pas si évident...

En tous cas, on peut en discuter :)

**Comment by loicknuchel** [comment]
Je pense que le mieux serait de faire des slides HTML qui font un appel à l'API pour remplir certains slides...
Je pense que quelqu'un d'autre que moi peut s'en charger... (je peux ajouter une API si nécessaire)

**Comment by sbouaked** [comment]
ahah flemmard :D 
@ygalescot t'es chaud ?

**Comment by loicknuchel** [comment]
Ah ben autant déléguer le peu qui est possible ^^ (j'assume le choix de Scala mais pour le front on peut partager ;) )

**Comment by ygalescot** [comment]
ok ça marche. Je rajoute ça à ma todo
